packages/ssr-web/deployment/fastboot-server.js

```diff
@@ -18,6 +21,14 @@ let server = new FastBootAppServer({
   },
   // This should be false for Twitter/Linkedin according to https://github.com/ember-fastboot/ember-cli-fastboot/tree/master/packages/fastboot-app-server#twitter-and-linkedin
```

Comment:
Not related to this PR, but I wonder if it's good to somehow provide `buildSandboxGlobals` to the local dev + the deployment server from the same source, so we don't have mysterious bugs show up if something changes.